leobot_description/robot/leobot.gazebo.xacro

```diff
@@ -237,4 +237,16 @@
         </sensor>
     </gazebo>
 
+    <gazebo>
+    <plugin name="object_controller" filename="libgazebo_ros_planar_move.so">
+      <commandTopic>/wheel_diff_drive_controller/cmd_vel</commandTopic>
```

Comment:
there topic are not needed. The only control topic is specified here https://github.com/ros-ukraine/leobot/blob/kinetic-devel/leobot_control/launch/control.launch#L11
Please read more about http://wiki.ros.org/twist_mux to find out how it works